apps/www/_blog/2024-03-26-semantic-image-search-amazon-bedrock.mdx

```diff
@@ -0,0 +1,241 @@
+---
+title: 'Implementing semantic image search with Amazon Titan and Supabase Vector'
+description: 'Implementing semantic image search with Amazon Titan and Supabase Vector in Python.'
+author: thor_schaeff
+image: getting-started/amazon-bedrock/amazon-bedrock-supabase-vecs.png
+thumb: getting-started/amazon-bedrock/amazon-bedrock-supabase-vecs.png
+categories:
+  - engineering
+tags:
+  - postgres
+  - developers
+  - ai
+date: '2024-03-26'
+toc_depth: 3
+---
+
+[Amazon Bedrock](https://aws.amazon.com/bedrock) is a fully managed service that offers a choice of high-performing foundation models (FMs) from leading AI companies like AI21 Labs, Anthropic, Cohere, Meta, Mistral AI, Stability AI, and Amazon via a single API, along with a broad set of capabilities you need to build generative AI applications with security, privacy, and responsible AI.
+
+[Amazon Titan](https://aws.amazon.com/bedrock/titan/) is a family of foundation models (FMs) for text and image generation, summarization, classification, open-ended Q&A, information extraction, and text or image search.
+
+In this guide we'll look at how we can get started with Amazon Bedrock and Supabase Vector in Python using the Amazon Titan multimodal model and the [vecs client](/guides/ai/vecs-python-client).
```

Comment:
Worth calling this one a `post` so that it sounds more bloggy?

```suggestion
In this post we'll look at how we can get started with Amazon Bedrock and Supabase Vector in Python using the Amazon Titan multimodal model and the [vecs client](/guides/ai/vecs-python-client).
```